### ISSUE TITLE
studio thumbnails use the default profile picture icon instead of the deafult studio thumbnail when there is no custom thumbnail set

### ISSUE THREAD

**Issue by MiniKiwiGeek** [scratchfoundation/scratch-www#8318]
### expected behavior

a studio with no custom thumbnail should possess the default studio thumbnail when visiting the studio (see screenshot 1).

### actual behavior

a studio with no custom thumbnail instead possesses the default profile picture used for user accounts with no custom profile picture (see screenshot 2).

### steps to reproduce

steps to reproduce the behavior:
1. go to 'https://scratch.mit.edu/mystuff' while logged into an account
2. click on 'my studios'
3. click on 'new studio'
4. see error
additional method:
1.) go to 'https://scratch.mit.edu'
2.) visit any studio that does not have a custom thumbnail, such as this one https://scratch.mit.edu/studios/35123720
3.) see error

### system details
_If the problem occurs in the browser, please provide operating system and browser e.g. Mac OS 10.11.6 Safari 10.0_

windows nt 10.0, chrome 124.0.0.0, no flash version detected, running on a z390 aorus ultra

**Screenshots**
intended behavior:
![A_studio](https://github.com/scratchfoundation/scratch-www/assets/111437961/e7fc5d86-7e20-4efa-b807-3499b00a0871)
actual behavior:
![wowie zowie](https://github.com/scratchfoundation/scratch-www/assets/111437961/50596c96-1844-4ead-ab30-ac6d668df514)

**Comment by PaperMarioFan-2022** [comment]
I don't think it is an error. They probably changed it because the old thumb didn't set right with the 3.0 pages, but that's just a guess.

**Comment by Olimon7661** [comment]
> I don't think it is an error. They probably changed it because the old thumb didn't set right with the 3.0 pages, but that's just a guess.

Actually, the old one looks more sleek. Besides, if they would set it to the default profile pic, why it since it is stretched horribly?
